_includes/documentation/documentation-version-band.html

```diff
@@ -26,9 +26,9 @@ <h3>Documentation for Version {{site.data.releases.operator[0].version}} - lates
       </p>
       <p>
         Strimzi Kafka operators Master<br>
-        <a href="{{site.baseurl}}/docs/operators/master/overview.html">Overview guide</a> | 
-        <a href="{{site.baseurl}}/docs/operators/master/quickstart.html">Quick Start guide</a> | 
-        <a href="{{site.baseurl}}/docs/operators/master/deploying.html">Deploying Strimzi</a> |
+        <a href="{{site.baseurl}}/docs/operators/master/overview.html">Overview guide</a> |
+        <a href="{{site.baseurl}}/docs/operators/master/quickstart.html">Quick Start guide</a> |
+        <a href="{{site.baseurl}}/docs/operators/master/deploying.html">Deploying and Upgrading Strimzi</a> |
```

Comment:
This seems to break the line. The book name is also used on more places. Maybe we should leave it to another PR where we can play a bit more with the design?

Reply:
Reverted